gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -362,49 +362,43 @@ gum_find_nearest_symbol_by_address (gpointer address,
                                     GumNearestSymbolDetails * nearest)
 {
   GHashTable * table;
-  gchar * name;
-  GList * keys, * cur;
-  gpointer current_address, nearest_address;
+  GumElfSymbolDetails * details;
+  GList * values, * cur;
+  GumElfSymbolDetails * current_symbol = NULL;
+  gboolean found = FALSE;
 
   table = gum_get_address_symbols ();
 
-  name = g_hash_table_lookup (table, address);
-  if (name != NULL)
+  details = (GumElfSymbolDetails *) g_hash_table_lookup (table, address);
```

Comment:
```suggestion
  details = g_hash_table_lookup (table, address);
```

(No need for cast when assigning from `void *`, here known as `gpointer`.)